discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -66,31 +68,31 @@ add rows, columns, and cells.
 
 Now you're familiar with the editor's regular mode. If you'd rather work with
 the content's HTML code, you can. To switch the editor to source view, select
-the *Source* icon (`</>`). Note that an icon (`abc`) for regular mode appears,
+the *Source* icon (`</>`). Note that an *abc* icon for regular mode appears,
 giving you the option to switch back to regular view. To satisfy your eyes,
 source view's moon icon and sun icon let you switch between a dark theme and a
 light theme. The built-in syntax coloring helps you identify HTML elements,
-regardless of the mode you're using.
+regardless of the mode you're using. 
 
-If you prefer to work in distraction-free mode, we've got you covered. You can
-switch to fullscreen mode by selecting the *Fullscreen* icon (bidirectional
-diagonally-crossed arrows). The *Edit Content* window appears, taking up the
-entire browser screen. The first icons above the editor allow you to arrange the
-HTML editor and rendered content area, horizontally or vertically. These dual
-views allow you to edit the HTML source and render it in real time. The last
-icon at the top of the content editor lets you toggle hiding the content
-rendering area, so you can use the entire browser space to work on the HTML
-code. Fullscreen view can really help you stay "in the zone" as you create
-awesome blog entries. 
+If you want to minimize distractions as you edit your blog entry's HTML code,
+we've got you covered. You can click the *Enlarge* icon
+(![Enlarge](../../images/enlarge-icon.png)) to open the HTML editor in a
+full-size *Edit Content* window. The first icons above the editor allow you to
+arrange the HTML editor and rendered content area, horizontally or vertically.
+These dual views allow you to edit the HTML source and render it in real time.
+The last icon at the top of the content editor lets you toggle hiding the
+content rendering area, so you can use the entire browser space to work on the
+HTML code. The enlarged HTML editor can really help you stay "in the zone" as
+you create awesome blog entries. 
```

Comment:
Again, I don't think the benefit of this is the pop up window. I'm not even sure that it's bigger than what you get without popping up the window. The benefit here is the dual pane view.